src/_guides/libraries/library-tour.md

```diff
@@ -1293,6 +1293,77 @@ await Future.wait([
 print('Done with all the long steps!');
 ```
 
+`Future.wait()` returns a future which will complete once all the provided
+futures have completed, either with their results, or with an error if any of
+the provided futures fail. 
```

Comment:
Worked around to shorten this, because a future completing is different than returning (I don't understand it much more than that...):

> `Future.wait()` returns a future which completes once all the provided
futures have completed. It completes either with their results,
or with an error if any of the provided futures fail.